code/modules/surgery/organs/organ_icon.dm

```diff
@@ -171,10 +171,10 @@ var/global/list/limb_icon_cache = list()
 			icon_file = 'icons/mob/human_races/robotic.dmi'
 		else
 			if(status & ORGAN_MUTATED)
-				icon_file = species.deform
+				icon_file = owner ? owner.deform : species.deform
 			else
 				// Congratulations, you are normal
-				icon_file = species.icobase
+				icon_file = owner ? owner.icobase : species.icobase
```

Comment:
Won't this break frankensteins by making all transplanted limbs look like the species of the owner, regardless of what species the source limb is? Should this `icobase` system be per-limb, instead?